Automattic/wp-calypso
# Jetpack Plans: Add Concierge back to the features listing

**Changes proposed in this Pull Request**
The program is coming off of being temporarily suspended, so we need to
add back any mentions of it to the Plans page and features listings.

**Testing instructions**
* Go to the Plans page for any Jetpack site.
* Confirm that the Concierge feature appears in the Premium list.
* Switch the view to “Monthly” billing.
* Confirm that the Concierge feature appears in the Premium list, in the same position.

![screen shot 2018-10-18 at 11 02 43 am](https://user-images.githubusercontent.com/625866/47164127-646c2780-d2c5-11e8-8eb1-83085d0c45ed.png)
